modules/serverless-deprecated-removed-features.adoc

```diff
@@ -13,37 +13,41 @@ For the most recent list of major functionality deprecated and removed within {S
 // OCP + OSD table
 ifdef::openshift-enterprise,openshift-dedicated[]
 .Deprecated and removed features tracker
-[cols="3,1,1,1,1",options="header"]
+[cols="3,1,1,1,1,1",options="header"]
 |====
-|Feature |1.20|1.21|1.22|1.23
+|Feature |1.20|1.21|1.22|1.23|1.24
```

Comment:
worth checking if 1.20 is still supported or if we want to remove the oldest version here, cc @nainaz

Reply:
same question for tech preview table, idk if it matters

Reply:
Did the change for the "GA & TP features" table only, per discussions. Thanks.